sample/F8/sessions

```diff
@@ -0,0 +1,1139 @@
+         "id": 4,
+         "name": "20 minutes session",
+         "length": "00.20"
+      },
+      "track": {
+         "id": 3,
+         "name": "ENGINEERING & OPEN SOURCE"
+      },
+      "long_abstract": "Learn about the unique scaling challenges encountered when developing Hack, the programming language behind Facebook.com. We’ll share the story of how we migrated from millions of lines of PHP, engineering through the problems that only come with a rapidly evolving codebase",
+      "speakers": [{
+         "organisation": "Facebook",
+         "id": 51,
+         "name": "Dwayne Reeves"
+      }]
+   },
+   {
+      "id": 32,
+      "subtitle": "moving-faster-optimizing-developer-experiences-of-facebook-engineers",
+      "title": "Moving Faster: Optimizing Developer Experiences of Facebook Engineers",
+      "long_abstract": "At Facebook, we help developers move fast by providing them with powerful tools. We'll take a look at how Facebook optimizes developer workflows today, and will explore the use of deep learning to power the next wave of advances.",
+      "end_time": "2017-04-18T16:20:00",
+      "microlocation": {
+         "id": 6,
+         "name": "F"
+      },
+      "start_time": "2017-04-18T16:00:00",
+      "comments": "",
+      "language": "",
+      "slides": "",
+      "audio": "",
+      "video": "",
+      "signup_url": "",
+      "state": "accepted",
+      "timezone": "UTC",
+      "session_type": {
+         "id": 4,
+         "name": "20 minutes session",
+         "length": "00.20"
+      },
+      "track": {
+         "id": 3,
+         "name": "ENGINEERING & OPEN SOURCE"
+      },
+      "speakers": [{
+            "organisation": "Facebook",
+            "id": 52,
+            "name": "Katie Coons"
+         },
+         {
+            "organisation": "Facebook",
+            "id": 53,
+            "name": "Kasper Kopec"
+         }
+      ]
+   },
+   {
+      "short_abstract": "",
+      "id": 33,
+      "subtitle": "explore-the-power-of-the-places-graph",
+      "title": "Explore the Power of the Places Graph",
+      "long_abstract": "Power your app with the Places Graph. We're providing free access to the same place data that powers Facebook, Instagram, and Messenger. Learn how to use data about 140M+ places around the world to create location aware app experiences.",
+      "end_time": "2017-04-18T16:20:00",
+      "start_time": "2017-04-18T16:00:00",
+      "microlocation": {
+         "id": 5,
+         "name": "G"
+      },
+      "comments": "",
+      "language": "",
+      "slides": "",
+      "audio": "",
+      "video": "",
+      "signup_url": "",
+      "state": "accepted",
+      "timezone": "UTC",
+      "session_type": {
+         "id": 4,
+         "name": "20 minutes session",
+         "length": "00.20"
+      },
+      "track": {
+         "id": 7,
+         "name": "FACEBOOK PLATFORM"
+      },
+      "speakers": [{
+            "organisation": "Facebook",
+            "id": 54,
```

Comment:
also please add all the info about speakers here in session json itself. i think we are not referring to speaker json to get data, instead we are getting it from here.